starter-code/src/app/app.module.ts

```diff
@@ -2,19 +2,34 @@ import { BrowserModule } from '@angular/platform-browser';
 import { NgModule } from '@angular/core';
 import { FormsModule } from '@angular/forms';
 import { HttpModule } from '@angular/http';
+import { Routes, RouterModule } from '@angular/router';
 
 import { AppComponent } from './app.component';
+import { MyHomeComponentComponent } from './components/my-home-component/my-home-component.component';
+import { MyMovieComponentComponent } from './components/my-movie-component/my-movie-component.component';
```

Comment:
The exercise says to name the page as `MyHomeComponent` and `MyMovieComponent` but since are pages, is a good practice name it as `HomePageComponent` and `MoviePageComponent`